src/Mvc/Mvc.Core/src/MvcCoreLoggerExtensions.cs

```diff
@@ -411,6 +412,11 @@ static MvcCoreLoggerExtensions()
             "No actions matched the current request. Route values: {RouteValues}",
             SkipEnabledCheckLogOptions);
 
+        _noActionDescriptors = LoggerMessage.Define(
+            LogLevel.Warning,
+            new EventId(1, "NoActionDescriptors"),
+            "No action descriptors detected, make sure your application is configured correctly or include additional Application parts. See more at https://aka.ms/aspnet/mvc/app-parts");
```

Comment:
```suggestion
            "No action descriptors found. This may indicate an incorrectly configured application or missing application parts. To learn more, visit https://aka.ms/aspnet/mvc/app-parts");
```

Reply:
Could you update the backing aka.ms URL to remove the locale: 

- https://docs.microsoft.com/en-us/aspnet/core/mvc/advanced/app-parts
+ https://docs.microsoft.com/aspnet/core/mvc/advanced/app-parts